src/App.js

```diff
@@ -147,7 +147,8 @@ function App() {
           path="/plant/:name"
           render={(routerProps) => <Plant {...routerProps}
           plant={selectedPlant}
-           deletePlant={deletePlant} 
+           deletePlant={deletePlant}
+           handleSubmit={handleUpdate} 
```

Comment:
The Plant.js page doesn't need the handleUpdate function. It gets passed to the PlantForm.js component